scan/lib/scan/report_collector.rb

```diff
@@ -56,6 +56,8 @@ def generate_commands(path, types: nil, output_file_name: nil)
     def determine_output_file_name(type)
       if @use_clang_report_name && type == "json-compilation-database"
         "compile_commands.json"
+      elsif type == "junit"
+        "report.xml"
```

Comment:
You're right... I will try to solve it by making new option so it won't break existing setups.